Ginger/GingerCoreNET/LiteDBFolder/LiteDbReportBase.cs

```diff
@@ -205,9 +225,24 @@ public class LiteDbActivity : LiteDbReportBase
     {
         public string ActivityGroupName { get; set; }
         public List<LiteDbAction> ActionsColl { get; set; }
+        //public List<Tuple<string, string>> ChildExecutableItemsCount { get; set; }
+        //public List<Tuple<string, string>> ChildExecutedItemsCount { get; set; }
+        //public List<Tuple<string, string>> ChildPassedItemsCount { get; set; }
+
+        public Dictionary<string, int> ChildExecutableItemsCount { get; set; }
```

Comment:
for Activity you save data only for Actions so why we need Dictionaries also here?